oeps/best-practices/oep-0051-bp-conventional-commits.rst

```diff
@@ -137,7 +137,11 @@ __ https://open-edx-proposals.readthedocs.io/en/latest/oep-0019-bp-developer-doc
 Footer
 ------
 
-Breaking changes must have a ``BREAKING CHANGE:`` footer in the body.  No other footers are specified.
+Breaking changes must have a ``BREAKING CHANGE:`` footer in the body.
+
+Commits may optionally include the ``the Co-authored-by:`` footer to indicate when the commit is the result of a collaboration.
```

Comment:
Nit: `include the ``the Co-authored-by:` has "the" twice; I don't think the second "the" belongs.